### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Name property is empty for "Deploy Web App dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1394]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Publish as Azure web app.
4. Observe the name property for the dialog 

**Actual Result:**
Name property is empty for the Dialog.

**Narrator Behaviour:**
selected, selection contains 16 items,
Scan Off
0% vertically scrolled

 **NVDA:**
dialog deploy web App
Select App Services to deploy to:
list

**Expected Result:**
Name property should be "Deploy Web App" dialog

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![deploy web app-dialog-narrator](https://user-images.githubusercontent.com/22956130/35683518-358d1e44-078a-11e8-844e-1b6e5e36dcf8.jpg)
![deploy web app-dialog-nvda](https://user-images.githubusercontent.com/22956130/35683526-397278d8-078a-11e8-8ae1-89edd986c16e.jpg)

**Comment by Eskibear** [comment]
ref #1407

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=533976

**Comment by jdneo** [comment]
* **Importance of experience**: minimal. Very low usage based on telemetry
* **Quality of experience**: not impacted. Users can still know the function of this dialog from the value field. (`Select App Service to deploy to` indicates that this dialog is used to deploy Web App)
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

![image](https://user-images.githubusercontent.com/22956130/35683526-397278d8-078a-11e8-8ae1-89edd986c16e.jpg)

**Comment by jdneo** [comment]
Can show the name property on my side.

Eclipse Version: Photon Release (4.8.0)
Plugin Version: 3.11.0

![image](https://user-images.githubusercontent.com/6193897/47150232-9357bd00-d308-11e8-8a4d-f79e655e4920.png)

**Comment by Tanushagottipati** [comment]
This issue is still reproducing in the current test pass

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749843
